Formula/chapel.rb

```diff
@@ -12,23 +12,36 @@ class Chapel < Formula
     sha256 x86_64_linux: "ca34ea32e25b7c9fea13bf25a09ac8b3151f45f8b8ac2dfeebfc0d18773783ba"
   end
 
+  depends_on "llvm@11"
   depends_on "python@3.9"
 
   def install
     libexec.install Dir["*"]
     # Chapel uses this ENV to work out where to install.
     ENV["CHPL_HOME"] = libexec
     # This is for mason
-    ENV["CHPL_REGEXP"] = "re2"
+    ENV["CHPL_RE2"] = "bundled"
 
     # Must be built from within CHPL_HOME to prevent build bugs.
     # https://github.com/Homebrew/legacy-homebrew/pull/35166
     cd libexec do
+      system "./util/printchplenv", "--all"
+      system "make"
+      # Need to let chapel choose target compiler with llvm
+      ENV["CHPL_HOST_CC"] = ENV["CC"]
+      ENV["CHPL_HOST_CXX"] = ENV["CXX"]
+      ENV.delete("CC")
+      ENV.delete("CXX")
+      system "./util/printchplenv", "--all"
       system "make"
       system "make", "chpldoc"
       system "make", "mason"
       system "make", "cleanall"
       rm_rf("third-party/llvm/llvm-src/")
+      rm_rf("third-party/gasnet/gasnet-src")
+      rm_rf("third-party/libfabric/libfabric-src")
+      rm_rf("third-party/fltk/fltk-1.3.5-source.tar.gz")
+      rm_rf("third-party/libunwind/libunwind-1.1.tar.gz")
```

Comment:
Can't it use the homebrew version?

Reply:
There's no need to use any of these so we are deleting them to save disk space.  The homebrew version of Chapel is designed for shared-memory execution only, so does not require gasnet or libfabric.  Macs have their own system unwind, so libunwind isn't required.  And we don't use fltk in the homebrew release.